Tests/Data/ThermoMechanics/iglu_axisymmetric_plane_strain_quad.prj

```diff
@@ -237,9 +236,9 @@
             <lis>-i bicgstab -p jacobi -tol 1e-11 -maxiter 10000</lis>
             <eigen>
                 <solver_type>BiCGSTAB</solver_type>
-                <precon_type>DIAGONAL</precon_type>
```

Comment:
A stronger preconditioner? Did the condition of the equation systems degrade in this PR?

Reply:
For this benchmark,  the condition of the equation systems is sensitive to the present change in the source code. I could either decrease the tolerance the convergence of the linear solver or change the preconditioner, and I chose the later. Note, the equation systems of TM by the monolithic scheme is unsymmetrical.